monitoring/instrumented.go

```diff
@@ -264,11 +256,3 @@ func (r *instrumentedRunnable) WaitRemovedMetric(metricName string, opts ...Metr
 
 	return errors.Newf("the metric %s is still exported by %s", metricName, r.Name())
 }
-
-func NewErrInstrumentedRunnable(name string, err error) InstrumentedRunnable {
```

Comment:
What's the replacement for this now? Trying to upgrade the version on the Thanos side

Reply:
It's InstrumentedRunnable{Runnable: e2e.NewError... (: